lib/account.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+
+class Account
+
+  attr_reader :forename, :surname, :username, :email, :password, :account_id
+
+  def initialize(forename:, surname:, username:, email:, password:, account_id:)
+    @forename = forename
+    @surname = surname
+    @username = username
+    @email = email
+    @password = password
+    @account_id = account_id
+  end
+
+  def self.create(forename:, surname:, username:, email:, password:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_challenge_test')
+    else
+      connection = PG.connect(dbname: 'chitter_challenge')
+    end
```

Comment:
I'd recommend you to look at the walkthrough of step 14 from last week's challenge. It explains how to extract a database setup object and make all these methods looking better.
https://github.com/makersacademy/course/blob/master/bookmark_manager/walkthroughs/14.md